sentry-samples/sentry-samples-spring-boot/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    id("org.springframework.boot") version "2.3.3.RELEASE"
+    id("io.spring.dependency-management") version "1.0.10.RELEASE"
+    kotlin("jvm")
+    kotlin("plugin.spring") version "1.3.72"
+}
+
+group = "com.example"
+version = "0.0.1-SNAPSHOT"
+java.sourceCompatibility = JavaVersion.VERSION_1_8
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-security")
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter")
+    implementation("org.jetbrains.kotlin:kotlin-reflect")
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")
```

Comment:
why did you skip the versioning? this could be a source of a broken example.
I'd reuse the `Config` class too

Reply:
Versions are managed by `io.spring.dependency-management` plugin - that's the way to use dependencies with Spring Boot.

Reply:
when you don't declare a version, you need either a `constraints` -> https://docs.gradle.org/current/userguide/single_versions.html#sec:declaring_without_version or `dependencyManagement` -> https://docs.spring.io/dependency-management-plugin/docs/current-SNAPSHOT/reference/html/
that's why I don't follow how the version is inferred, maybe the plugin does it internally, but I didn't find the docs for it.

also, I don't think that the `io.spring.dependency-management` also manages the versioning of the kotlin deps.

Reply:
eg `implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8")` could become `implementation(kotlin(Config.kotlinStdLib))`

Reply:
I also don't know exactly how it works under the hood (one reason I prefer Maven over Gradle is it's simplicity), but there is no need to define anything extra to let this plugin resolve dependencies. By not specifying versions we are sure that the imported libraries are compatible with Spring Boot.

Reply:
wfm